apps/endpoint-registry/sql/create_asset_administration_shells.sql

```diff
@@ -7,13 +7,14 @@ CREATE TABLE public.asset_administration_shells
     CONSTRAINT asset_administration_shells_pkey PRIMARY KEY ("aasId"),
     CONSTRAINT "assetId" FOREIGN KEY ("assetId")
         REFERENCES public.assets ("assetId") MATCH SIMPLE
-        ON UPDATE NO ACTION
-        ON DELETE NO ACTION
```

Comment:
the relation between aas and asset is aas n -- 1 asset. 
update and delete on an aas should not affect the asset.

Reply:
Good point, thanks